.github/workflows/ci.yml

```
@@ -162,12 +162,13 @@ jobs:
         ports:
         - 5432:5432
       elasticsearch:
-        image: elasticsearch:6.8.23
+        image: elasticsearch:7.17.1
```

```suggestion
        image: elasticsearch:7
```

Actually, just going to pin the major version here so we don't drift out of date.

---

Should have checked if they actually provide this image...